### DISCUSSION TITLE
Mastodon Lists

### DISCUSSION THREAD

**Discussion by hermann-san** [mastodon/mastodon#18667] [General]
Hi,
there is a feature called "lists" and I'm having some trouble
- understanding what it is
- Finding a description and instructions on the [help page](https://mastodon.help/) on that feature
- adding accounts I follow to a list. When I edit a list, it doesn't show me an account that I could add, even though I follow 2 accounts.

Thanks for your hints.
I'm using Firefox 101 on Linux

**Comment by nightpool** [comment]
https://mastodon.help is a third party resource that is unaffiliated with the Mastodon developers. We cannot give support for it. Mastodon lists work identically to how Twitter private lists work. 

> When I edit a list, it doesn't show me an account that I could add, even though I follow 2 accounts.

I don't know how this could possibly happen. It sounds like an error in your server's setup

**Reply by hermann-san** [reply]
thanks for your response
I've tried 4 different public servers. 
social.dev-wiki.de
mstdn.social   (large server)
social.tchncs.de  (large server)
ruhr.social

in all of those, the people I'm following do not appear in the list.  The list is empty on all 3 profiles. On the right hand side the accounts I'm following, as an example. 
There are not prerequisites for lists, right? e.g. minimum number of account to follow, type of accounts or alike
![Auswahl_008](https://user-images.githubusercontent.com/7679654/173865200-24d97eca-c7cc-4609-98b3-5ec999adf665.png) ![grafik](https://user-images.githubusercontent.com/7679654/173865428-17e3dd80-c6fd-4a7b-adf9-70bcf6524a58.png)

**Reply by hermann-san** [reply]
I've found out the trick. 
I was assuming I only need to type the first few letters and it would automatically show me the names. Or I would click on the magnifying glass. But nothing happened when I did that. So the trick is to hit enter after typing a least one (correct first) letter

IMO not perfectly intuitive :o)
but at least i've now found out how it works.
